libtransmission/peer-mgr.cc

```diff
@@ -832,6 +837,7 @@ class tr_swarm
         }
         else
         {
+            ++stats.known_peer_from_count[info_this.from_first()];
```

Comment:
```suggestion
            ++stats.known_peer_from_count[nh.mapped().from_first()];
```

From https://en.cppreference.com/w/cpp/container/unordered_map/extract:

> Pointers and references to the extracted element remain valid, but cannot be used while element is owned by a node handle: they become usable if the element is inserted into a container.